src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/noLineWrap/InputNoLineWrapBad.java

```diff
@@ -0,0 +1,28 @@
+package com.puppycrawl.tools.
```

Comment:
i do not understand what you want to say here....could you please elaborate a little........